pennylane/qchem/vibrational/localize_modes.py

```diff
@@ -0,0 +1,248 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module contains functions to localize normal modes."""
+
+import warnings
+
+import numpy as np
+import scipy
+
+import pennylane as qml
+
+# pylint: disable=dangerous-default-value, too-many-statements
+
+
+def _mat_transform(u, qmat):
+    r"""Returns the rotated displacement vectors matrix for a given rotation unitary u
+    and displacement vectors matrix qmat.
+
+    Args:
+       u (TensorLike[float]): unitary rotation matrix
+       qmat (TensorLike[float]): matrix of displacement vectors
+
+    Returns:
+       (TensorLike[float]): rotated matrix of displacement vectors
```

Comment:
Indentation here and in many other places are 3 spaces instead of four. Double check that your code editor uses 4 spaces for tabs, and you can just quickly unindent and re-indent to fix all of the docstrings. Make sure they are all 4 spaces before merging.